Slack is the preferred way to get help (#12133)
apache/druid: README.md

### Community

Community support is available on the `#druid` channel in the Apache Slack team. Please send an email to 

> druid-user@googlegroups.com to get an invite to slack channel. 
> You must send the request from same email-id that slack invitation has to be sent to. 

Reviewer: ```suggestion
druid-user@googlegroups.com to get an invite to the Slack channel. 
Send the request from the same email address that you want the Slack invitation sent to. 
```
